config/engines.yml

```diff
@@ -55,7 +55,17 @@ duplication:
   description: Structural duplication detection for Ruby, Python, JavaScript, and PHP.
   community: false
   enable_regexps:
+    - \.js$
+    - \.jsx$
+    - \.php$
```

Comment:
@GordonDiggs Are those extensions common enough to include by default or should we leave it as an opt in? If we add those two other extensions here, we should probably add them for ratings as well.